### DISCUSSION TITLE
Solis offline

### DISCUSSION THREAD

**Discussion by stan916** [wills106/homeassistant-solax-modbus#457] [Q&A]
I’m running this integration on my Pi4 on HAOSS and using 2 waveshare poe devices as client and server so I can have the integration working alongside the solis cloud app simultaneously. Everything worked fine until June 15th when everything in the solis app and web portal is offline for a few hours then briefly online for 15-20 mins then offline again. HA data works fine all the time, and if I plug the solis datalogger directly into the inverter the solis data is fine (but obvs not HA data then). Anyone any ideas what might be wrong? Have rolled back HA core and supervisor to versions I was running prior to June 15th but this makes no difference and it was all running perfectly before then with uninterrupted data on HA and solis app.

**Comment by alienatedsec** [comment]
@stan916 
Have you checked the latest guidelines from the project? https://github.com/alienatedsec/solis-ha-modbus-cloud 

It also feels like both of your Waveshare devices don't communicate very well. Any IP configurations and conflicts would be my best bet.
